src/bundle/Resources/public/scss/_forms.scss

```diff
@@ -108,3 +108,35 @@ form:not(.form-inline) {
 .ez-content-create[readonly] {
     pointer-events: none;
 }
+
+.ibexa-form-error {
+    display: flex;
+    font-family: $font-family-sans-serif;
+    font-size: calculateRem(12px);
```

Comment:
`$ibexa-text-font-size-small`?

Or maybe you could create some mixin because you repeat 3 time
```
font-family: $font-family-sans-serif;
font-size: calculateRem(12px);
letter-spacing: 0.12px;
line-height: calculateRem(18px);
```

Reply:
I was thinking about the same. But I'm not sure what I can combine to make it reusable. 
Maybe 2 mixins:
- one with: font-family and letter-spacing
- and the other with: font-size and line-height 
?